model/project_ref.go

```diff
@@ -316,6 +321,20 @@ const (
 	maxBatchTime             = 153722867 // math.MaxInt64 / 60 / 1_000_000_000
 )
 
+type ProjectRefSection string
+
+const (
+	ProjectRefGeneralSection        = "general"
+	ProjectRefAccessSection         = "access"
+	ProjectRefVariablesSection      = "variables"
+	ProjectRefGithubSection         = "github"
```

Comment:
This is technically both a github and commit queue section on the UI.

Reply:
Yeah; I thought this simplified it since commit queue might just be github PRs in the future but eh I'll be clearer.

Reply:
ah I hear that! thanks!